deployment/README.md

```diff
@@ -2,12 +2,16 @@
 
 ## Ansible Deploy
 
-This implementation supports OpenStack dynamic inventory by using [Ansible 2.4 OpenStack inventory plugin](https://docs.ansible.com/ansible/devel/plugins/inventory/openstack.html).
+This implementation supports:
+
+OpenStack dynamic inventory by using [Ansible 2.4 OpenStack inventory plugin](https://docs.ansible.com/ansible/devel/plugins/inventory/openstack.html).
 To install it's dependencies with `pip`:
 ```bash
 pip install -r ansible/pip-requirements.txt
 ```
 
+AWS EC2 dynamic inventory [EC2 Inventory](https://github.com/ansible/ansible/blob/cdf8f29e70c15598d0a92c8f17841e58d240f94b/contrib/inventory/ec2.py)
```

Comment:
This does not look good to me in the middle of nothing. Please see https://github.com/aeternity/epoch/blob/98c62239df9e6c88a3146a77134b431c3a676711/deployment/README.md

Plase take a time to better structure the documentation and split the supported inventories in separate sections.
Also the link should point to the [inventory documentation](https://docs.ansible.com/ansible/devel/plugins/inventory/aws_ec2.html) and the source version (tag) should be pointed in the ec2.py file e.g. https://github.com/ansible/ansible/blob/v2.5.0rc3/contrib/inventory/ec2.py